core/src/main/java/hudson/slaves/SlaveComputerUtil.java

```diff
@@ -0,0 +1,30 @@
+package hudson.slaves;
+
+import hudson.FilePath;
+import hudson.remoting.Channel;
+import hudson.remoting.VirtualChannel;
+import jenkins.model.Jenkins;
+
+public final class SlaveComputerUtil {
+    private SlaveComputerUtil() {}
+    /**
+     * Obtains a {@link VirtualChannel} that allows some computation to be performed on the master.
+     * This method can be called from any thread on the master, or from agent (more precisely,
+     * it only works from the remoting request-handling thread in agents, which means if you've started
+     * separate thread on agents, that'll fail.)
+     *
+     * @return null if the calling thread doesn't have any trace of where its master is.
+     * @since XXX
```

Comment:
```suggestion
     * @since TODO
```

per https://github.com/jenkinsci/jenkins/blob/3b85c0a877d97c4c8b5b24605974cce52223a4b0/update-since-todo.sh#L13